app/main.py

```diff
@@ -1,7 +1,38 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+    city = os.getenv("CITY")
+    if not api_key:
+        raise ValueError("API_KEY is not set")
+    if not city:
+        raise ValueError("CITY is not set")
+
+    URL = "http://api.weatherapi.com/v1/current.json?"
+    response = requests.get(URL + f"key={api_key}&q={city}")
+    if response.status_code == 200:
+        return (
+            f"{response.json()["location"]["name"]}/"
+            f"{response.json()["location"]["country"]} "
+            f"{response.json()["location"]["localtime"]} "
+            f"Weather: {response.json()["current"]["temp_c"]} Celsius, "
+            f"{response.json()["current"]["condition"]["text"]}"
```

Comment:
The string formatting in the return statement is incorrect due to the use of double quotes inside double quotes. This will cause a syntax error. Consider using single quotes for the string literals or escaping the double quotes.